server/graphql/types/src/types/stock_line.rs

```diff
@@ -105,6 +106,19 @@ impl StockLineNode {
     pub async fn barcode(&self) -> Option<&str> {
         self.stock_line.barcode()
     }
+
+    pub async fn vvm_status(&self, ctx: &Context<'_>) -> Result<Option<VVMStatusNode>> {
+        let loader = ctx.get_loader::<DataLoader<VVMStatusLoader>>();
```

Comment:
Given that the vvm_status_id is Optional, you can return early here with something like:

`
if self.row().vvm_status_id.is_none() {
   return Ok(None)
}
`